### ISSUE TITLE
Změna značení Lagrangiánů

### ISSUE THREAD

**Issue by mpicek** [ufal/npfl129#80]
Možná by bylo fajn zavést nové písmenko pro Lagrangiány, protože se v přednáškách vyskytují skoro furt. Jelikož loss je značena L a lagraniány taktéž L, tak se to občas plete. Vždy je z kontextu jasné, o co jde, přesto mě to občas trochu zmátlo :)

Navrhuji pro Lagrangiány použít psací L, někde je ale v přednáškách značena loss také pomocí psacího L (teď mě rychle napadá např. hinge loss), tak na to dát bacha při úpravě.

Bude to fajn vylepšení, ale rozhodně to není nutné podle mě.

**Comment by foxik** [comment]
Jo, díky, to je dobrý nápad, a i když je snažím říkat, co je Lagrangián, mít jednotné značení věci určitě prospěje. Zatím to nechám otevřené a během pár dní to provedu; ale bod za komunitní práci dostáváte rovnou :-)
